book/graphics.md

```diff
@@ -337,19 +337,25 @@ enough text, not all of the lines will fit on the screen. We want
 users to *scroll*\index{scroll} the page to look at different parts of it.
 
 ::: {.further}
-In English text, you can't wrap to the next line in the middle of a
-word (without hyphenation at least), but in Chinese that's mostly not a problem. Mostly, but not
-always! <span lang="zh">开关</span> means "button" but is composed of
-<span lang="zh">开</span> "on" and <span lang="zh">关</span> "off". A
-line break between them would be confusing, because you'd read "on
-off" instead of "button". The [ICU library][icu], used by both Firefox
-and Chrome, [uses dynamic programming][icu-wb] to guess phrase
-boundaries based on a [word frequency table][cjdict].
+In English text, you can't wrap to the next line in the middle of a word
+(without hyphenation at least), but in Chinese that's the default,
+even for words made up of multiple characters (for example,
+开关 meaning "switch" is composed of 开 "on" and 关 "off", but it's just fine
```

Comment:
```suggestion
<span lang="zh">开关</span> meaning "switch" is composed of <span lang="zh">开</span> "on" and <span lang="zh">关</span> "off", but it's just fine
```

The language tags are important because it otherwise uses the Japanese presentation by default.